dashboard/app/assets/javascripts/components/report_abuse_form.jsx

```diff
@@ -18,16 +18,24 @@ window.dashboard.ReportAbuseForm = (function (React) {
    */
   var AgeDropdown = React.createClass({
     propTypes: {
-      age: React.PropTypes.string
+      age: React.PropTypes.oneOfType([
+        React.PropTypes.string,
+        React.PropTypes.number
+      ])
     },
 
     render: function () {
       var style = $.extend({}, {width: DROPDOWN_WIDTH}, this.props.style);
 
+      var age = this.props.age;
+      if (typeof(age) === 'number') {
+        age = age.toString();
```

Comment:
My thinking was that if we were to unintentionally pass in some sort of object, we would then get an error that showed the object instead of [Object object], though thinking about it line 39 is just going to call toString() on this.props.age anyways, so I think maybe you're right.